roles/hosted_engine_setup/tasks/initial_clean.yml

```diff
@@ -29,6 +29,10 @@
     args:
       warn: false
     environment: "{{ he_cmd_lang }}"
+  - name: Get stats of a file
```

Comment:
Please try to be more specific in the task name, maybe something like `Verify abrt-action-save-package-data file`.